system-int-test/src/cmr/system_int_test/utils/search_util.clj

```diff
@@ -247,6 +247,16 @@
                  :throw-exceptions true
                  :connection-manager (s/conn-mgr)})))
 
+(defn get-autocomplete-suggestions
+  "Executes a query to the autocomplete endpoint with the given value and returns the results."
+  ([]
+   (client/get (url/autocomplete-url)))
+  ([term]
+   (client/get (url/autocomplete-url term)))
+  ([term types-list]
```

Comment:
add optional `opts` to your args so you can pass in things like `headers` or `throw-exceptions false`